### PR TITLE
Miscellaneous documentation fixes/changes

### PR DESCRIPTION
**Pull request by rahulporuri**
This PR makes a number of miscennaleous documenation fixes and changes. These were made when reviewing the entire documentation. Most of the changes are style changes in the example code included, some are spelling corrections and some fix rendering issues with sphinx.

### CODE REVIEW

File: docs/source/user_manual/chaco_tutorial.rst [hunk 1]
```diff
@@ -727,10 +755,12 @@ By default, an ``Enum`` trait will be displayed as a drop-down. In the
 constructor, we create a dictionary that maps the data names to actual
 numpy arrays::
 
-    # jn is the Bessel function
-    self.data = {"jn0": jn(0, x),
-                 "jn1": jn(1, x),
-                 "jn2": jn(2, x)}
+            # jn is the Bessel function or order n
```
Comment (aaronayres35) [inline docs/source/user_manual/chaco_tutorial.rst:758]: nit: "of" not "or"

```suggestion
            # jn is the Bessel function of order n
```

File: docs/source/user_manual/containers.rst [hunk 2]
```diff
@@ -54,8 +54,9 @@ E.g., this code::
         # Create a vertical container containing two horizontal containers
         h_container1 = HPlotContainer()
         h_container2 = HPlotContainer()
-        outer_container = VPlotContainer(h_container1, h_container2,
-                                            stack_order="top_to_bottom")
+        outer_container = VPlotContainer(
+          h_container1, h_container2, stack_order="top_to_bottom"
```
Comment (aaronayres35) [inline docs/source/user_manual/containers.rst:58]: 2 space indent
```suggestion
            h_container1, h_container2, stack_order="top_to_bottom"
```

File: docs/source/user_manual/containers.rst [hunk 3]
```diff
@@ -120,17 +121,20 @@ horizontal container::
 
         # Create the plot
         plot = Plot(data)
-        plot.plot(("index", "value", "color"), type="cmap_scatter",
-                  color_mapper=jet)
+        plot.plot(
+          ("index", "value", "color"), type="cmap_scatter", color_mapper=jet
```
Comment (aaronayres35) [inline docs/source/user_manual/containers.rst:125]:
```suggestion
            ("index", "value", "color"), type="cmap_scatter", color_mapper=jet
```

File: docs/source/user_manual/containers.rst [hunk 4]
```diff
@@ -120,17 +121,20 @@ horizontal container::
...
-                            resizable='v',
-                            width=30,
-                            padding=20)
+        colorbar = ColorBar(
+          index_mapper=LinearMapper(range=colormap.range),
+          color_mapper=colormap,
+          orientation='v',
+          resizable='v',
+          width=30,
+          padding=20,
```
Comment (aaronayres35) [inline docs/source/user_manual/containers.rst:136]:
```suggestion
            index_mapper=LinearMapper(range=colormap.range),
            color_mapper=colormap,
            orientation='v',
            resizable='v',
            width=30,
            padding=20,
```

File: docs/source/user_manual/containers.rst [hunk 5]
```diff
@@ -226,15 +230,19 @@ The complete code looks like this:
...
             # Create a GridContainer to hold all of our plots: 2 rows, 3 columns
-            container = GridPlotContainer(shape=(2,3),
-                                          spacing=(10,5),
-                                          valign='top',
-                                          bgcolor='lightgray')
+            container = GridPlotContainer(
+              shape=(2,3),
+              spacing=(10,5),
+              valign='top',
+              bgcolor='lightgray',
```
Comment (aaronayres35) [inline docs/source/user_manual/containers.rst:244]:
```suggestion
                shape=(2,3),
                spacing=(10,5),
                valign='top',
                bgcolor='lightgray',
```

File: docs/source/user_manual/containers.rst [hunk 6]
```diff
@@ -246,13 +254,14 @@ The complete code looks like this:
                 pd.set_data(data_name, jn(i,x))
 
                 plot = Plot(pd)
-                plot.plot(('index', data_name),
-                          color=COLOR_PALETTE[i],
-                          line_width=3.0)
+                plot.plot(
+                  ('index', data_name),
+                  color=COLOR_PALETTE[i],
+                  line_width=3.0,
```
Comment (aaronayres35) [inline docs/source/user_manual/containers.rst:260]:
```suggestion
                    ('index', data_name),
                    color=COLOR_PALETTE[i],
                    line_width=3.0,
```

File: docs/source/user_manual/containers.rst [hunk 7]
```diff
@@ -320,23 +331,31 @@ full data: ::
             pd = ArrayPlotData(index=x, value=y)
 
             zoomable_plot = Plot(pd)
-            zoomable_plot.plot(('index', 'value'),
-                               name='external', color='red', line_width=3)
+            zoomable_plot.plot(
+              ('index', 'value'),
+              name='external',
+              color='red',
+              line_width=3,
```
Comment (aaronayres35) [inline docs/source/user_manual/containers.rst:338]:
```suggestion
                ('index', 'value'),
                name='external',
                color='red',
                line_width=3,
```

File: docs/source/user_manual/containers.rst [hunk 8]
```diff
@@ -320,23 +331,31 @@ full data: ::
...
+            )
 
             # Attach tools to the plot
-            zoom = ZoomTool(component=zoomable_plot,
-                            tool_mode="box", always_on=False)
+            zoom = ZoomTool(
+              component=zoomable_plot,
+              tool_mode="box",
+              always_on=False,
```
Comment (aaronayres35) [inline docs/source/user_manual/containers.rst:345]:
```suggestion
                component=zoomable_plot,
                tool_mode="box",
                always_on=False,
```

File: docs/source/user_manual/containers.rst [hunk 9]
```diff
@@ -320,23 +331,31 @@ full data: ::
...
-            inset_plot.set(resizable = '',
-                           bounds = [250, 150],
-                           position = [450, 350],
-                           border_visible = True
-                           )
+            inset_plot.set(
+              resizable='',
+              bounds=[250, 150],
+              position=[450, 350],
+              border_visible=True,
```
Comment (aaronayres35) [inline docs/source/user_manual/containers.rst:357]:
```suggestion
                resizable='',
                bounds=[250, 150],
                position=[450, 350],
                border_visible=True,
```

File: docs/source/user_manual/containers.rst [hunk 10]
```diff
@@ -434,16 +453,29 @@ Rendering order
...
-6. :attr:`overlay`: Legends, selection regions, and other tool-drawn visual
-    elements
+
+6. **selection**: Selected content are rendered above normal plot elements to
+   make them stand out. This can be disabled by setting :attr:`use_selection`
+   to False (default).
+
+7. **border**: Plot borders
+
+8. **annotation**: Lines and text that are conceptually part of the "plot" but
```
Comment (aaronayres35) [inline docs/source/user_manual/containers.rst:474]: annotation bullet appears twice

File: docs/source/user_manual/plot_types.rst [hunk 11]
```diff
@@ -381,8 +381,12 @@ The preferred way to do this is using the factory method
 
     w, h = image_source.get_width(), image_source.get_height()
     index = GridDataSource(np.arange(w), np.arange(h))
-    index_mapper = GridMapper(range=DataRange2D(low=(0, 0),
-                                                high=(w-1, h-1)))
+    index_mapper = GridMapper(
+      range=DataRange2D(
+        low=(0, 0),
+        high=(w-1, h-1),
+      )
```
Comment (aaronayres35) [inline docs/source/user_manual/plot_types.rst:388]:
```suggestion
        range=DataRange2D(
            low=(0, 0),
            high=(w-1, h-1),
        )
```

File: docs/source/user_manual/tutorial_hyetograph.rst [hunk 12]
```diff
@@ -27,11 +27,11 @@ the soil) a plot shows the intensity vs. time hyetograph plots.
...
 =================
 
-To run this demo you must have Chaco and its dependencies installed,
+To run this demo you must have Chaco and its dependencies installed:
 
- * Traits
- * TraitsGUI
- * Enable
+* Traits
+* TraitsGUI
```
Comment (aaronayres35) [inline docs/source/user_manual/tutorial_hyetograph.rst:33]: TraitsUI

File: docs/source/user_manual/tutorial_van_der_waal.rst [hunk 13]
```diff
@@ -31,38 +31,54 @@ You must have Chaco and its dependencies installed:
 * TraitsGUI
```
Comment (aaronayres35) [inline docs/source/user_manual/tutorial_van_der_waal.rst:31]: TraitsUI

File: docs/source/user_manual/tutorial_van_der_waal.rst [hunk 14]
```diff
@@ -31,38 +31,54 @@ You must have Chaco and its dependencies installed:
...
     class Data(HasTraits):
+
         volume = Array
+
         pressure = Array
+
         attraction = Range(low=-50.0,high=50.0,value=0.0)
-        totVolume = Range(low=.01,high=100.0,value=0.01)
+
+        tot_volume = Range(low=.01,high=100.0,value=0.01)
```
Comment (aaronayres35) [inline docs/source/user_manual/tutorial_van_der_waal.rst:76]: nit: spaces after commas

File: docs/source/user_manual/basic_elements/overlays.rst [hunk 15]
```diff
@@ -71,15 +73,19 @@ A :class:`~chaco.axis.MinorPlotAxis` should be added along with a
...
 
-    x_major_axis = PlotAxis(orientation='bottom',
-                      title='My x axis',
-                      mapper=plot.x_mapper,
-                      component=plot)
+    x_major_axis = PlotAxis(
+      orientation='bottom',
+      title='My x axis',
+      mapper=plot.x_mapper,
+      component=plot,
```
Comment (aaronayres35) [inline docs/source/user_manual/basic_elements/overlays.rst:80]:
```suggestion
        orientation='bottom',
        title='My x axis',
        mapper=plot.x_mapper,
        component=plot,
```

File: docs/source/user_manual/basic_elements/overlays.rst [hunk 16]
```diff
@@ -71,15 +73,19 @@ A :class:`~chaco.axis.MinorPlotAxis` should be added along with a
...
+    )
     plot.underlays.append(x_major_axis)
 
-    x_minor_axis = MinorPlotAxis(orientation='bottom',
-                      mapper=plot.x_mapper,
-                      component=plot)
+    x_minor_axis = MinorPlotAxis(
+      orientation='bottom',
+      mapper=plot.x_mapper,
+      component=plot,
```
Comment (aaronayres35) [inline docs/source/user_manual/basic_elements/overlays.rst:87]:
```suggestion
        orientation='bottom',
        mapper=plot.x_mapper,
        component=plot,
```

**Review by aaronayres35 — APPROVED** [review]
Changes LGTM

A few nit picky comments, most of which are about instances of using a 2 not 4 space indent

File: docs/source/user_manual/tutorial_hyetograph.rst (hunk 12 above)
Comment (rahulporuri) [inline docs/source/user_manual/tutorial_hyetograph.rst:33]:
```suggestion
* TraitsUI
```

File: docs/source/user_manual/tutorial_van_der_waal.rst (hunk 13 above)
Comment (rahulporuri) [inline docs/source/user_manual/tutorial_van_der_waal.rst:31]:
```suggestion
* TraitsUI
```

File: docs/source/user_manual/tutorial_van_der_waal.rst (hunk 14 above)
Comment (rahulporuri) [inline docs/source/user_manual/tutorial_van_der_waal.rst:76]:
```suggestion
        tot_volume = Range(low=.01, high=100.0, value=0.01)
```

File: docs/source/user_manual/tutorial_van_der_waal.rst [hunk 17]
```diff
@@ -31,38 +31,54 @@ You must have Chaco and its dependencies installed:
...
     from chaco.chaco_plot_editor import ChacoPlotItem
     from numpy import arange
 
     class Data(HasTraits):
+
         volume = Array
+
         pressure = Array
+
         attraction = Range(low=-50.0,high=50.0,value=0.0)
```
Comment (rahulporuri) [inline docs/source/user_manual/tutorial_van_der_waal.rst:74]:
```suggestion
        attraction = Range(low=-50.0, high=50.0, value=0.0)
```

**Pull request merged**